sdk/src/main/java/com/uid2/EUIDManager.kt

```diff
@@ -0,0 +1,107 @@
+package com.uid2
+
+import android.content.Context
+import com.uid2.UID2Manager.Companion
+import com.uid2.UID2Manager.Companion.APPLICATION_ID_DEFAULT
+import com.uid2.UID2Manager.Companion.UID2_API_URL_DEFAULT
+import com.uid2.network.DefaultNetworkSession
+import com.uid2.network.NetworkSession
+import com.uid2.storage.FileStorageManager
+import com.uid2.storage.FileStorageManager.Store.EUID
+import com.uid2.storage.StorageManager
+import com.uid2.utils.InputUtils
+import com.uid2.utils.Logger
+import com.uid2.utils.TimeUtils
+import kotlinx.coroutines.Dispatchers
+
+public class EUIDManager {
+
+    public companion object {
+        public data class Environment private constructor(
```

Comment:
This should probably be a sealed interface, rather than a data class with hardcoded types, e.g.

```
public sealed interface Environment {
  public val serverUrl: String

  public data class London(
    override val serverUrl = "https://prod.euid.eu/v2"
  ) : Environment
}
```